docker-compose.yml

```diff
@@ -171,13 +178,34 @@ services:
 
   localstack:
     container_name: localstack
+    entrypoint: /bin/bash -c # overwrites entrypoint
+    command:
+      - |
+        echo 'editing startup script'
+
+        # Remove last 3 lines of startup script so we can run some setup
+        sed -i -n -e :a -e '1,3!{P;N;D;};N;ba' /usr/local/bin/docker-entrypoint.sh
+
+        # run_startup_scripts function in forground insted of backround
+        echo 'run_startup_scripts' >> /usr/local/bin/docker-entrypoint.sh
+
+        bash '/usr/local/bin/docker-entrypoint.sh'
+        echo 'settting up buckets'
+        aws --endpoint-url=http://localhost:4572 s3 mb s3://filestore-bucket
+        aws --endpoint-url=http://localhost:4572 s3api put-bucket-acl --bucket filestore-bucket --acl public-read
+        aws --endpoint-url=http://localhost:4572 s3 mb s3://external-filestore-bucket
+        aws --endpoint-url=http://localhost:4572 s3api put-bucket-acl --bucket external-filestore-bucket --acl public-read
+
+        tail -qF /tmp/localstack_infra.log /tmp/localstack_infra.err
```

Comment:
We need run our setup after startup but by default after the the script gives up no chance to do so as it tails the logs forever. 

This removes that tail until we are ready

Reply:
cant say i'm a fan of this. can't we boot everything as normal then call a script to setup for tests?

Reply:
That on my radar, this way all the docker-compose cmds work as expected. But yeah doing that would be less complex

Reply:
replaced with a script